profiler/options_test.go

```diff
@@ -37,18 +37,20 @@ func TestOptions(t *testing.T) {
 	})
 
 	t.Run("WithAPIKey", func(t *testing.T) {
+		var validKey = "12345678901234567890123456789012"
```

Comment:
```suggestion
		validKey := "12345678901234567890123456789012"
```

Reply:
Also, maybe we can have just one global constant `testAPIKey`

Reply:
Great suggestions.

Writing the tests revealed an issue with non-ASCII glyphs returning true by unicode.IsLower(), so this already paid off.

Also added a simple positive and negative check for Profiler instantiation.